AllReadyApp/Web-App/AllReady.UnitTest/Areas/Admin/Controllers/EventAdminControllerTests.cs

```diff
@@ -30,11 +30,22 @@ public class EventAdminControllerTests
             await TaskFromResultZero;
         }
 
-        [Fact(Skip = "NotImplemented")]
+        [Fact]
         public async Task DetailsReturnsHttpNotFoundResultWhenEventIsNull()
         {
-            // delete this line when starting work on this unit test
-            await TaskFromResultZero;
+            var imageService = new Mock<IImageService>();
+            var mediator = new Mock<IMediator>();
+            mediator.Setup(x => x.SendAsync(It.IsAny<CampaignSummaryQuery>())).ReturnsAsync(new CampaignSummaryModel());
+
+            var eventDetailModelValidator = new Mock<IValidateEventDetailModels>();
+            eventDetailModelValidator.Setup(x => x.Validate(It.IsAny<EventDetailModel>(), It.IsAny<CampaignSummaryModel>()))
+                .Returns(new List<KeyValuePair<string, string>>());
+
```

Comment:
you can delete the above three lines that deal with creating a mock of eventDetailModelValidator b/c the Details action method does not invoke any members of the interface. You can pass in null for IValidateEventDetailModels